Nitrox.Test/Serialization/Helper/NitroxAutoFaker.cs

```diff
@@ -0,0 +1,42 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Reflection;
+using AutoBogus;
+using NitroxModel.DataStructures.GameLogic;
+
+namespace Nitrox.Test.Serialization.Helper;
+
+public class NitroxAutoFaker<TType> : AutoFaker<TType> where TType : class
+{
+    public NitroxAutoFaker() : this(new Dictionary<Type, Type[]>()) { }
+
+    public NitroxAutoFaker(Dictionary<Type, Type[]> subtypesByBaseType)
+    {
+        Configure(newBinder =>
+        {
+            newBinder.WithBinder(new PacketAutoBinder(subtypesByBaseType))
```

Comment:
We can't use generics here (bc the NitroxAutoBinderBase shouldn't be allowed to be initialized with `new()`). Therefore I am suggesting that the binder should be made a parameter.

Reply:
Do you mean you want the binder type to be a type parameter on `NitroxAutoFaker`?

Reply:
Yes